src/Editor.ts

```diff
@@ -177,7 +180,7 @@ export default class Editor extends EventDispatcher {
       change = new TextChange(this.doc, change);
     }
     const old = this.doc;
-    const doc = old.apply(change);
+    const doc = old.apply(change, undefined, this.throwOnError);
```

Comment:
Previously, no error was thrown if a delta went beyond the end of a document, so I didn't want to start throwing errors always, but I did want to provide the option to have errors thrown to find bugs in your system.